README.md

```diff
@@ -1,5 +1,9 @@
 # dnsname plugin
 
+## IMPORTANT
+
+This repository is no longer actively maintained. Our new development efforts are happening in [netavark](https://github.com/containers/netavark/) and [aardvark-dns](https://github.com/containers/aardvark-dns/) that should provide a better alternative.
```

Comment:
Any time my brain sees the word "new", it yells at me.

Sorry to be a nag, but would you mind adding "As of 2023"? E.g.
>As of 2023, this repository is ...